ftest/web-ui/webdriver/test/features/clipboard.feature

```diff
@@ -1,33 +1,71 @@
-    | File       | File3            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
-    | File       | File4            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
-    | File       | File5            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
-    And I login as "Administrator"
-    When I browse to the document with path "/default-domain/Src"
-    And I select all child documents
-    Then I can see the selection toolbar
-    And I can add selection to clipboard
+      | doctype   | title | nature  | subjects           | coverage        | creator | path                | collections | tag | file |
+      | Workspace | Src   | booklet | sciences/astronomy | europe/Portugal | BJones  | /default-domain     |             |     |      |
+      | Workspace | Dest  | booklet | sciences/astronomy | europe/Portugal | BJones  | /default-domain     |             |     |      |
+      | File      | File1 | booklet | sciences/astronomy | europe/Portugal | BJones  | /default-domain/Src |             |     |      |
+      | File      | File2 | booklet | sciences/astronomy | europe/Portugal | BJones  | /default-domain/Src |             |     |      |
+      | File      | File3 | booklet | sciences/astronomy | europe/Portugal | BJones  | /default-domain/Src |             |     |      |
+    And user "John" exists in group "members"
+    And I login as "John"
+    And I have a clean clipboard
+
+  Scenario: Add documents to the clipboard
+    Given I browse to the document with path "/default-domain/Src"
+    And I select the "File1" document
+    And I can see the selection toolbar
+
+    When I can add selection to clipboard
+    And I click the "clipboard" button
+    Then I can see the clipboard has "1" items
+    And I can see the clipboard has "File1" document
+
+  Scenario: Remove documents from the clipboard
+    Given I have document with path "/default-domain/Src/File1" on clipboard
+
     When I click the "clipboard" button
-    Then I can see the clipboard has "5" items
-    When I browse to the document with path "/default-domain/Dest"
+    Then I can see the clipboard has "File1" document
+
+    When I click remove button for "File1" document
+    Then I can see the clipboard has "0" items
+
+  Scenario Outline: <action> from clipboard
+    Given I have document with path "/default-domain/Src/File1" on clipboard
+    And I have document with path "/default-domain/Src/File2" on clipboard
+    And I have document with path "/default-domain/Src/File3" on clipboard
+    And I have permission ReadWrite for the document with path "/default-domain/Src"
+    And I have permission ReadWrite for the document with path "/default-domain/Dest"
```

Comment:
Since we now have two of these in a row, I'd consider using a data table here as well, something like `Given I have permission <permission> for the following documents:`.